publiccode.yml

```diff
@@ -0,0 +1,36 @@
+publiccodeYmlVersion: "0.4"
+
+name: OSRD
+
+url: "https://github.com/OpenRailAssociation/osrd"
+
+platforms:
+  - web
+
+developmentStatus: development
+
+softwareType: "standalone/web"
+
+description:
+  en:
+    shortDescription: >
+          OSRD is an open source web application for railway infrastructure design, capacity analysis, timetabling and simulation.
+    documentation: https://osrd.fr/en/docs/
+  fr:
+    shortDescription: >
+          OSRD est une application web open source pour la conception d'infrastructures ferroviaires, l'analyse de la capacité, la programmation et la simulation.
+    documentation: https://osrd.fr/fr/docs/
+
+legal:
+  license: LGPL-3.0-or-later
```

Comment:
I didn't find any other place where we state that versions > 3.0 are okay. Should this be "LGPL-3.0" instead?